cekit/descriptor/resource.py

```diff
@@ -80,6 +80,12 @@ def __ne__(self, other):
             return not self['name'] == other['name']
         return NotImplemented
 
+    @staticmethod
+    def empty_descriptor(default_value, container=""):
```

Comment:
By looking at the name of this method is not easy to get what it actually does.

How about creating a property instead? It could use an abstract method to generate default values which would be implemented in the other XYZResource classes. Or something similar? WDYT?

Reply:
Thinking on this, its kind of pointless, its wrapping a single log line. Eliminating it might be more logical